drivers/ieee802154/ieee802154_cc13xx_cc26xx.c

```diff
@@ -0,0 +1,731 @@
+/*
+ * Copyright (c) 2019 Brett Witherspoon
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_LEVEL CONFIG_IEEE802154_DRIVER_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(ieee802154_cc13xx_cc26xx);
+
+#include <device.h>
+#include <errno.h>
+#include <misc/byteorder.h>
+#include <net/ieee802154_radio.h>
+#include <net/ieee802154.h>
+#include <net/net_pkt.h>
+#include <random/rand32.h>
+#include <string.h>
+#include <sys_io.h>
+
+#include <driverlib/aon_rtc.h>
+#include <driverlib/osc.h>
+#include <driverlib/prcm.h>
+#include <driverlib/rf_ieee_mailbox.h>
+#include <driverlib/rfc.h>
+#include <inc/hw_ccfg.h>
+#include <inc/hw_fcfg1.h>
+
+#if defined(CONFIG_SOC_SERIES_CC13X2_CC26X2)
```

Comment:
Is this ifdef needed, isn't the driver only built for these SoCs?

Reply:
I also had support for the CC13x0 and CC26x0 series and removed that for this PR but left the ifdefs. I can remove and reintroduce them if I add support for older series later.